lib/pure/collections/sequtils.nim

```diff
@@ -571,6 +571,26 @@ template filterIt*(s, pred: untyped): untyped =
     if pred: result.add(it)
   result
 
+since (1, 1):
+  template containsIt*(s, pred: untyped): bool =
```

Comment:
This template doesn't work because it has `return`. Also anyIt exists https://nim-lang.org/docs/sequtils.html#anyIt.t%2Cuntyped%2Cuntyped